modules/tinymce/src/plugins/table/test/ts/atomic/UtilTest.ts

```diff
@@ -2,37 +2,9 @@ import { describe, it } from '@ephox/bedrock-client';
 import { assert } from 'chai';
 import * as fc from 'fast-check';
 
-import { addPxSuffix, isPercentage, isPixel, removePxSuffix } from 'tinymce/plugins/table/core/Util';
+import { addPxSuffix, removePxSuffix } from 'tinymce/plugins/table/core/Utils';
 
 describe('atomic.tinymce.plugins.table.core.UtilTest', () => {
-  it('isPercentage', () => {
-    assert.isFalse(isPercentage(''), 'Empty string is false');
-    assert.isFalse(isPercentage('%'), 'Single % string is false');
-    assert.isTrue(isPercentage('10%'), 'Percentage string is true');
-    assert.isTrue(isPercentage('10.125%'), 'Percentage with decimal string is true');
-
-    fc.assert(fc.property(fc.float(1, 100), (n) => {
-      assert.isTrue(isPercentage(n + '%'), 'Arbitrary float with percent string is true');
-      assert.isFalse(isPercentage(n + ''), 'Number string is false');
-      assert.isFalse(isPercentage(n + 'px'), 'Pixel string is false');
-      assert.isFalse(isPercentage(n + '%' + n), 'String containing % string is false');
-    }));
-  });
-
-  it('isPixel', () => {
-    assert.isFalse(isPixel(''), 'Empty string is false');
-    assert.isFalse(isPixel('px'), 'Single px string is false');
-    assert.isTrue(isPixel('10px'), 'Pixel string is true');
-    assert.isTrue(isPixel('10.125px'), 'Pixel with decimal string is true');
-
-    fc.assert(fc.property(fc.float(1, 100), (n) => {
-      assert.isTrue(isPixel(n + 'px'), 'Arbitrary float with px string is true');
-      assert.isFalse(isPixel(n + ''), 'Number string is false');
-      assert.isFalse(isPixel(n + '%'), 'Percent string is false');
-      assert.isFalse(isPixel(n + 'px' + n), 'String containing px string is false');
-    }));
-  });
```

Comment:
Theses tests look to have just been deleted? That doesn't seem right as they should have been retained somewhere based on where things moved.

- [ ] Ensure these tests are retained/moved to a relevant location

Reply:
Have fixed this and added the deleted tests to `atomic/table/TableUtilsTest.ts` in core